en/mkdocs.yml

```diff
@@ -70,7 +70,7 @@ nav:
           - Perform Root Cause Analysis: observability/perform-root-cause-analysis.md
           - Performance Analysis: observability/analyze-performance.md
     - Insights:
-          - Derive Insights: insights/derive-insights.md
+          - View API Insights: insights/derive-insights.md
```

Comment:
Would be good to change the file name too since we generally follow that practice.

Reply:
Corrected